APR-WrathOfTheLichKing/WrathOfTheLichKing_Alliance.lua

```diff
@@ -1,3 +1,1640 @@
+        {	-- Step 94
+             ["CRange"] = 12724,
+             ["TT"] = {
+                  ["y"] = 1649.3,
+                  ["x"] = -5976.4,
+             },
+             ["Trigger"] = {
+                  ["y"] = 1649.3,
+                  ["x"] = -5976.4,
+             },
+             ["Range"] = 15.06,
+        },
+        {	-- Step 95
+             ["Qpart"] = {
+                  [12724] = {
+                       ["1"] = "1",
+                  },
+             },
+             ["TT"] = {
+                  ["y"] = 1649.3,
+                  ["x"] = -6017.7,
+             },
+             ["Trigger"] = {
+                  ["y"] = 1649.3,
+                  ["x"] = -6017.7,
+             },
+             ["Range"] = 9.52,
+        },
+        {	-- Step 96
+             ["Done"] = {
+                  12724,
+             },
+             ["TT"] = {
+                  ["y"] = 1404.5,
+                  ["x"] = -5826.5,
+             },
+        },
+        {	-- Step 97
+             ["PickUp"] = {
+                  12725,
+             },
+             ["TT"] = {
+                  ["y"] = 1399.0,
+                  ["x"] = -5826.5,
+             },
+        },
+        {	-- Step 98
+             ["Done"] = {
+                  12725,
+             },
+             ["TT"] = {
+                  ["y"] = 1654.0,
+                  ["x"] = -6039.9,
+             },
+        },
+        {	-- Step 99
+             ["PickUp"] = {
+                  12727,
+             },
+             ["TT"] = {
+                  ["y"] = 1654.0,
+                  ["x"] = -6039.9,
+             },
+        },
+        {	-- Step 100
+             ["Qpart"] = {
+                  [12727] = {
+                       ["1"] = "1",
+                  },
+             },
+             ["TT"] = {
+                  ["y"] = 1648.4,
+                  ["x"] = -6040.4,
+             },
+             ["Trigger"] = {
+                  ["y"] = 1648.4,
+                  ["x"] = -6040.4,
+             },
+             ["Range"] = 5.03,
+        },
+        {	-- Step 101
+             ["Done"] = {
+                  12727,
+             },
+             ["TT"] = {
+                  ["y"] = 1398.0,
+                  ["x"] = -5826.8,
+             },
+        },
+        {	-- Step 102
+             ["PickUp"] = {
+                  12738,
+             },
+             ["TT"] = {
+                  ["y"] = 1398.0,
+                  ["x"] = -5826.8,
+             },
+        },
+        {	-- Step 103
+             ["Done"] = {
+                  12738,
+             },
+             ["TT"] = {
+                  ["y"] = 1375.8,
+                  ["x"] = -5713.9,
+             },
+        },
+        {	-- Step 104
+             ["PickUp"] = {
+                  12742,
+             },
+             ["TT"] = {
+                  ["y"] = 1375.8,
+                  ["x"] = -5713.9,
+             },
+             ["PickUpDB"] = {
+                  12743,

```

Comment:
Insert "12742," -- the DB array has to include the questID in the step. I missed that initially and fixed it already in the spreadsheet.